data/common.yaml

```diff
@@ -0,0 +1,2 @@
+---
+nomad::init_style: "%{facts.service_provider}"
```

Comment:
can you move this to the init.pp? That allows puppet-strings to pick it up

Reply:
i actually already did, but forgot to remove it here